added carparklist table migration file

sameer-heidi opened the pull request:
migration for creating carpark table: carparklist

sameer-heidi committed:
added carparklist table migration file

an-heidi commented on CityDBMigrations/Migrations/20250519153100_AddCarparkList.cs:
```diff
@@ -0,0 +1,61 @@
+            // Create the carparklist table
+            Execute.Sql(@"
+                CREATE TABLE carparklist (
+                    id VARCHAR(100) NOT NULL,
```
add primary key here itself

an-heidi commented on CityDBMigrations/Migrations/20250519153100_AddCarparkList.cs:
```diff
@@ -0,0 +1,61 @@
+
+            // Insert seed data
+            Execute.Sql(@"
+                INSERT INTO carparklist 
```
no need to add insertion query into the migration as this will only be used in one project

an-heidi commented on CityDBMigrations/Migrations/20250519153100_AddCarparkList.cs:
```diff
@@ -0,0 +1,61 @@
+                    roof VARCHAR(50) DEFAULT NULL,
+                    lon VARCHAR(200) DEFAULT NULL,
+                    lat VARCHAR(200) DEFAULT NULL,
+                    monday VARCHAR(100) DEFAULT NULL,
```
add it in date format so that if in the future we need to use that, we could

an-heidi commented on CityDBMigrations/Migrations/20250519153100_AddCarparkList.cs:
```diff
@@ -0,0 +1,61 @@
+                    id VARCHAR(100) NOT NULL,
+                    name VARCHAR(100) DEFAULT NULL,
+                    address VARCHAR(200) DEFAULT NULL,
+                    `free` VARCHAR(50) DEFAULT NULL,
```
why is this added as a string?

an-heidi commented on CityDBMigrations/Migrations/20250519153100_AddCarparkList.cs:
```diff
@@ -0,0 +1,61 @@
+                    capacity VARCHAR(50) DEFAULT NULL,
+                    ampel VARCHAR(50) DEFAULT NULL,
+                    roof VARCHAR(50) DEFAULT NULL,
+                    lon VARCHAR(200) DEFAULT NULL,
```
use longitude and latitude itself

an-heidi commented on CityDBMigrations/Migrations/20250519153100_AddCarparkList.cs:
```diff
@@ -0,0 +1,61 @@
+                    address VARCHAR(200) DEFAULT NULL,
+                    `free` VARCHAR(50) DEFAULT NULL,
+                    capacity VARCHAR(50) DEFAULT NULL,
+                    ampel VARCHAR(50) DEFAULT NULL,
```
I recommend using an enum or a traffic_status table so that it will be easier to query later on. And be easy with constant variables

sameer-heidi committed:
updated CarparkList file and added ParkingTimings list

sameer-heidi committed:
updated timestamp

an-heidi commented on CityDBMigrations/Migrations/20250519154730_AddParkingTimings.cs:
```diff
@@ -0,0 +1,30 @@
+            Execute.Sql(@"
+                CREATE TABLE `parking_timings` (
+                    `day` TINYINT NOT NULL,
+                    `parking_id` VARCHAR(100) NOT NULL,
```
let's  use parkingId to follow the current coding pattern

an-heidi commented on CityDBMigrations/Migrations/20250519154730_AddParkingTimings.cs:
```diff
@@ -0,0 +1,30 @@
+namespace DatabaseMigrations.Migrations
+{
+    [Migration(20250519154730)]
+    public class CreateParkingTimingsTable : Migration
```
do the same for carparklist too

an-heidi commented on CityDBMigrations/Migrations/20250519153100_AddCarparkList.cs:
```diff
@@ -0,0 +1,31 @@
+        public override void Up()
+        {
+            Execute.Sql(@"
+                CREATE TABLE `carparklist` (
```
and name the file accordingly

an-heidi replied:
and the class as well

sameer-heidi committed:
updated table,column and file name

sameer-heidi committed:
moved carparking migrations files

sameer-heidi committed:
Add migration scripts for Mullkalender database tables

- Created `mullkalender_push_devices` table to store device information.
- Created `mullkalender_push_device_streets` table to link devices with streets.
- Created `mullkalender_push_device_waste_types` table to associate waste types with device streets.
- Created `mullkalender_notification_log` table to log notifications sent to devices.

devfiveheidi committed:
Merge pull request #195 from KODI-Kommunen-Digital/feat/fcmpushnotificationwastecalender

Add migration scripts for Mullkalender database tables
